docs/linking_to_external_code/integrity_checking.md

```diff
@@ -26,8 +58,16 @@ Collaborator on another machine -- in a freshly cloned project tree:
 ```shell
 # Download the project's dependencies into the machine's cache, integrity
 # checking each resource.
-deno cache -r --lock=lock.json src/deps.ts
+deno cache --reload --lock=lock.json src/deps.ts
 
 # Done! You can proceed safely.
 deno test --allow-read src
 ```
+
+Or alternatively
+
+```shell
+# Run src/deps.ts, validating dependencies against lock.json and updating
+# lock.json with any new ones.
+deno run --lock=lock.json --lock-write src/deps.ts
+```
```

Comment:
This section isn't necessary. The way to do this is written earlier with:
```sh
deno cache --lock=lock.json --lock-write src/deps.ts
```
I think updating the lock file as a side-effect of the `run` command doesn't make for a good example, and is especially confusing to be repeated here.

Reply:
Thanks for the feedback @nayeemrmn, I've updated the content to be more clear on best practice for updating lock files.  However, this does make me wonder why `--lock-write` is an option for `deno run` if it's not desirable behaviour?

Reply:
I mostly mean it doesn't make for a good example because it does too many things.

To answer your question though, it's logical for `deno run` to just inherit all of `deno cache`'s flags (just as it inherits its behaviour) rather than selecting them based on an exemplary workflow.